rest/data/aliases.go

```diff
@@ -26,6 +26,7 @@ func FindProjectAliases(projectId, repoId string, aliasesToAdd []restModel.APIPr
 			aliasModels = append(aliasModels, a)
 		}
 	}
+	// TODO <ticket_num>: modify to correctly merge aliases based on type
```

Comment:
Can we create a ticket and link it here?

Reply:
yeah wanted to hold on to until this change was approved 👍  will make now